src/app.ts

```diff
@@ -12,9 +13,41 @@ import completions from './api/completions';
 // Create the app
 const app = express();
 
+// Config settings for Kinde
+if (process.env.ENVIRONMENT !== 'dev') {
+  const { setupKinde } = require('@kinde-oss/kinde-node-express');
+
+  const config = {
```

Comment:
Environment variables required for Kinde should be added to documentation. Not their values, but I think they should at least be referenced